examples/release_highlights/plot_release_highlights_1_5_0.py

```diff
@@ -0,0 +1,182 @@
+# ruff: noqa
+"""
+=======================================
+Release Highlights for scikit-learn 1.5
+=======================================
+
+.. currentmodule:: sklearn
+
+We are pleased to announce the release of scikit-learn 1.5! Many bug fixes
+and improvements were added, as well as some key new features. Below we
```

Comment:
```suggestion
and improvements were added, as well as some new key features. Below we
```

Reply:
I let you see with @betatim for this one :) (https://github.com/scikit-learn/scikit-learn/pull/29007#discussion_r1605144118)

Reply:
He‘s native English.